<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>703</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>797</width>
    <height>703</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fachrichtungen</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="7">
    <widget class="QGroupBox" name="gbFachrichtungen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>779</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Fachrichtungen</string>
     </property>
     <widget class="QLineEdit" name="leFachrichtungabk">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFachrichtungbez">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblFachrichtung">
      <property name="geometry">
       <rect>
        <x>236</x>
        <y>43</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fachrichtung:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblBezeichnung">
      <property name="geometry">
       <rect>
        <x>237</x>
        <y>103</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bezeichnung:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLJahre">
      <property name="geometry">
       <rect>
        <x>253</x>
        <y>161</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lehrjahre:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbLehrjahre">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>158</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>3.000000000000000</double>
      </property>
      <property name="maximum">
       <double>4.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btSuchen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Suchen</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</normaloff>../../_Symbole/UsedSymbols/searchmagnifierinterfacesymbol1_79893.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btSpeichern">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Speichern</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/save_3621.ico</normaloff>../../_Symbole/UsedSymbols/save_3621.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btLoeschen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Löschen</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</normaloff>../../_Symbole/UsedSymbols/seo-social-web-network-internet_262_icon-icons.com_61518.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btAbbrechen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Hauptmenü</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</normaloff>../../_Symbole/UsedSymbols/log-out_icon-icons.com_50106.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btExcel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</normaloff>../../_Symbole/UsedSymbols/microsoft_excel_icon_132212.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="btCSV">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Report</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</normaloff>../../_Symbole/UsedSymbols/reporting_noun_report_document_file_icon_148360.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="btHilfe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Hilfe</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../_Symbole/UsedSymbols/info_13213.ico</normaloff>../../_Symbole/UsedSymbols/info_13213.ico</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QTableWidget" name="twAnzeige">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>400</height>
      </size>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QLabel" name="lab_INFO">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leFachrichtungabk</tabstop>
  <tabstop>leFachrichtungbez</tabstop>
  <tabstop>sbLehrjahre</tabstop>
  <tabstop>btSuchen</tabstop>
  <tabstop>btSpeichern</tabstop>
  <tabstop>btAbbrechen</tabstop>
  <tabstop>btLoeschen</tabstop>
  <tabstop>btExcel</tabstop>
  <tabstop>btCSV</tabstop>
  <tabstop>btHilfe</tabstop>
  <tabstop>twAnzeige</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
